lms/static/sass/shared/_fields.scss

```diff
@@ -0,0 +1,102 @@
+// lms - shared - fields
+// ====================
+
+
+.u-field {
+    padding: $baseline 0;
+    border-bottom: 1px solid $gray-l5;
+    border: 1px dashed transparent;
+
+    &.mode-placeholder:hover {
+        border: 1px dashed $m-blue-d1;
```

Comment:
By `$link-blue` do you mean `$link-color` coz i'm not able to find variable with this name.